.packit.yaml

```diff
@@ -68,3 +62,18 @@ jobs:
     tmt_plan: /integration
     skip_build: true
     identifier: integration-test
+
+  - job: propose_downstream
+    trigger: release
+    dist_git_branches:
+      - fedora-all
+
+  - job: koji_build
+    trigger: commit
+    dist_git_branches:
+      - fedora-all
+
+  - job: bodhi_update
+    trigger: commit
+    dist_git_branches:
+      - fedora-all
```

Comment:
I think this should be `fedora-branched`. That's because rawhide updates are created automatically. Having `fedora-all` will likely cause packit to complain (IIUC by filing an upstream issue). @lachmanfrantisek correct me if I'm wrong.

Everything else LGTM

Reply:
Thanks @lsm5! That makes sense to me. Updated.